frontend/src/pages/Login.jsx

```
@@ -0,0 +1,100 @@
+import { useState } from "react"
+import { FaEye, FaEyeSlash, FaUser } from "react-icons/fa";
+import { MdEmail, MdLock } from "react-icons/md";
+import { useNavigate } from "react-router-dom";
+
+const sampleUser = {
+    userEmail: "shukriasul5@gmail.com",
+    userPassword: "Shukria@123"
+}
+
+export default function Login() {
+    const [passwordType, setPasswordType] = useState("password")
+    const [email, setEmail] = useState("")
+    const [password, setPassword] = useState("")
+    const [loading, setLoading] = useState(false)
+
+    const handlePasswordVisibility = () => {
+        setPasswordType(prev => prev === "password" ? "text" : "password")
+    }
+    const navigate = useNavigate()
+    const handleLogin = async (e) => {
+        e.preventDefault()
+        if (!email || !password) return alert("Please fill all fields!")
+        setLoading(true)
+        try {
+            await new Promise((resolve) => setTimeout(resolve, 2000))
+            if (sampleUser.userEmail === email && sampleUser.userPassword === password) {
+                alert("Success! 🎉")
+                navigate("/")
+            } else {
+                alert("Invalid email or password!")
+            }
+        } finally {
```

you missed the catch block for error handling now it is silently fail

---

Thanks for this feedback, it is a very important point. Added the catch block.